src/DependencyManagement/WorkerEnvironment.cs

```diff
@@ -16,6 +16,8 @@ internal static class WorkerEnvironment
         private const string ContainerName = "CONTAINER_NAME";
         private const string LegionServiceHost = "LEGION_SERVICE_HOST";
 
+        private static readonly DateTime PowerShellSDKDeprecationDate = new DateTime(2026, 11, 10);
```

Comment:
We'll need to remember to keep updating this in the future. Perhaps we should add a reminder item to https://github.com/Azure/azure-functions-powershell-worker/blob/dev/upgrade_ps_sdk_instructions.md

Reply:
Added create_new_worker_instructions.md to document steps that must be taken when updating to a new PS 7 minor version (like 7.6). 
This file includes a note to update the Managed Dependencies shutoff date. It is likely we will find many new checklist items for this file as we start the 7.6 worker next year.